keyboards/andean_condor/keymaps/default/keymap.c

```diff
@@ -0,0 +1,13 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_TAB,  KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,         KC_Y,  KC_U,  KC_I,     KC_O,    KC_P,         KC_BACKSPACE,
+        KC_LCTL, KC_A,  KC_S,  KC_D,  KC_F,  KC_G,         KC_H,  KC_J,  KC_K,     KC_L,    KC_SEMICOLON, KC_ENTER, 
+        KC_LSFT, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,         KC_N,  KC_M,  KC_COMMA, KC_DOT,  KC_SLASH,     LSFT(KC_SEMICOLON),
+        KC_1,    KC_2,  KC_3,  KC_4,  KC_5,  KC_6,         KC_7,  KC_8,  KC_9,     KC_0,    KC_MINUS,     KC_EQUAL
```

Comment:
The keymap should somewhat represent the keyboard and whats expressed in `info.json`

```
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐                ┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│TA││Q ││W ││E ││R ││T │                │Y ││U ││I ││O ││P ││BS│
└──┘└──┘└──┘└──┘└──┘└──┘                └──┘└──┘└──┘└──┘└──┘└──┘
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐                ┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│CT││A ││S ││D ││F ││G │                │H ││J ││K ││L ││: ││RE│
└──┘└──┘└──┘└──┘└──┘└──┘                └──┘└──┘└──┘└──┘└──┘└──┘
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐                ┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│SH││Z ││X ││C ││V ││B │                │N ││M ││, ││. ││/ ││; │
└──┘└──┘└──┘└──┘└──┘└──┘                └──┘└──┘└──┘└──┘└──┘└──┘
                      ┌──┐┌──┐    ┌──┐┌──┐
                      │lt││lt│    │rt││rt│
                      └──┘└──┘    └──┘└──┘
              ┌──┐┌──┐┌──┐┌──┐    ┌──┐┌──┐┌──┐┌──┐
              │lb││lb││lb││lb│    │rb││rb││rb││rb│
              └──┘└──┘└──┘└──┘    └──┘└──┘└──┘└──┘
```

I would expect the keymap to look more like the following (it would require re-ordering the `info.json` layout items)

```c
        KC_TAB,  KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,         KC_Y,  KC_U,  KC_I,     KC_O,    KC_P,         KC_BACKSPACE,
        KC_LCTL, KC_A,  KC_S,  KC_D,  KC_F,  KC_G,         KC_H,  KC_J,  KC_K,     KC_L,    KC_SEMICOLON, KC_ENTER, 
        KC_LSFT, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,         KC_N,  KC_M,  KC_COMMA, KC_DOT,  KC_SLASH,     LSFT(KC_SEMICOLON),
                                      KC_5,  KC_6,         KC_7,  KC_8,
                        KC_1,  KC_2,  KC_3,  KC_4,         KC_9,  KC_0,  KC_MINUS, KC_EQUAL
```

Reply:
Done.